src/components/AppLayout/index.tsx

```diff
@@ -19,7 +19,7 @@ const Container = styled.div`
 const HeaderWrapper = styled.nav`
   height: 52px;
   width: 100%;
-  z-index: 2;
+  z-index: 1299;
```

Comment:
1300 is the z-index of the backdrop, so we want the header below it but above the sidebar (to drop the shadow on it).